src/styles/utils/_functions.scss

```
@@ -0,0 +1,5 @@
+@import "../utils/variables";
+
+@function containerWidth($card-width, $container__card-margin--right) {
```

function name should start with a verb
```suggestion
@function calcWidth($card-width, $container__card-margin--right) {
```